### ISSUE TITLE
Make win32 installer ask for Pidgin installation if not found

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#114]
```
What steps will reproduce the problem?
1. Install PidginPortable
2. Attempt to install msn-pecan
3. Receive error that it couldn't find Pidgin

What is the expected output? What do you see instead?
Expected output would be to allow choice for where Pidgin is installed when
it isn't detected in a known location.

What version of the product are you using? On what operating system?
msn-pecan 0.0.17
PidginPortable 2.5.3
Windows XP (Unable to determine windows version, work computer that is
locked down.)

Please provide any additional information below.
Despite it being a work computer I have been able to install many
applications and this is not a case of security causing this. The installer
likely can not detect where the installation folder is because I am setup
with PidginPortable in a non-standard pidgin directory.

Due to not allowing me to select where Pidgin is located it just errors out
saying it "Failed to find Pidgin installation."
```

Original issue reported on code.google.com by `XWraithanX` on 12 Jan 2009 at 5:30

**Comment by GoogleCodeExporter** [comment]
```
Workaround:

Open the installer program with 7zip portable (may work with other compression 
apps)
then extract [$35]\plugins\libmsn-pecan.dll to the App\Pidgin\plugins subdir on 
your
portable pidgin installation.
```

Original comment by `bruno.ti...@gmail.com` on 12 Jan 2009 at 9:04

**Comment by GoogleCodeExporter** [comment]
```
This works, thank you.

It would awesome if the installer was changed, or the dll up for direct 
download for
those of us that need it in that form.
```

Original comment by `XWraithanX` on 12 Jan 2009 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
Would it really make that much of a difference? It's just a matter of 
extracting a
.exe file.
```

Original comment by `felipe.contreras` on 12 Jan 2009 at 9:58

**Comment by GoogleCodeExporter** [comment]
```
I already did it but it seems like a rather poor installer if it can't adapt to 
even
a simple change like that. Also, would it really make that much of a difference 
to
keep the latest version of the dll up for direct download? 

It seems like it is more bothersome to have an installer that automagically 
takes
care of stuff rather than just putting a dll in the appropriate folder. You're
plugin, up to you, just seems like a poor choice to not support any non standard
configurations at all.
```

Original comment by `XWraithanX` on 13 Jan 2009 at 12:37

**Comment by GoogleCodeExporter** [comment]
```
XWraithanX: you are free to improve the installer.

I'll think about adding a separate .dll for the next version.
```

Original comment by `felipe.contreras` on 13 Jan 2009 at 3:35

**Comment by GoogleCodeExporter** [comment]
```
I've looked into the installer file and there is this:

ReadRegStr $0 HKLM "Software\pidgin" ""
IfFileExists "$0\pidgin.exe" cont

It would be easy to do a similar search for pidginportable, right? I just need 
the
applicable strings.

If you provide them in less than one week it will go into 0.0.18 :)
```

Original comment by `felipe.contreras` on 5 Feb 2009 at 10:15
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
```
Too late, 0.0.18 has been released, but perhaps for 0.0.19? I just need you to 
check
the registry.
```

Original comment by `felipe.contreras` on 15 Feb 2009 at 11:36

**Comment by GoogleCodeExporter** [comment]
```
I checked the registry. There are no visible entries for pidgin portable other 
than
those on the MRU lists. Sorry.
```

Original comment by `bruno.ti...@gmail.com` on 21 Feb 2009 at 1:01

**Comment by GoogleCodeExporter** [comment]
```
bruno: huh? So there's no way to find out the installed location?

If that's the case then there's nothing msn-pecan's installer can do, that 
needs to
be fixed first in pidgin portable.
```

Original comment by `felipe.contreras` on 21 Feb 2009 at 1:10
- Changed state: **Waiting**

**Comment by GoogleCodeExporter** [comment]
```
The whole point of my post was that the installer should ASK for a location if 
pidgin
is not found. 

The best thing about the PortableApps is that they leave no trace, so you can 
throw
them on a flash drive and use them anywhere. 
```

Original comment by `XWraithanX` on 21 Feb 2009 at 3:11

**Comment by GoogleCodeExporter** [comment]
```
What is the problem with using a extractor utility like 7zip?

Lowering the priority but I'm thinking on setting this as the first WontFix.
```

Original comment by `felipe.contreras` on 15 Mar 2009 at 7:48
- Added labels: **Priority-Low**

**Comment by GoogleCodeExporter** [comment]
```
The better question is, what is the problem with changing the installer to 
allow your
to select where to install to?
```

Original comment by `XWraithanX` on 15 Mar 2009 at 9:59

**Comment by GoogleCodeExporter** [comment]
```
How do you suggest to do that?
```

Original comment by `felipe.contreras` on 15 Mar 2009 at 10:35

**Comment by GoogleCodeExporter** [comment]
```
i think i understood...
adding a separate .dll would be nice but changing the installer would be great 
for 
those who can't download but can at least execute
```

Original comment by `vinicius...@gmail.com` on 15 Apr 2009 at 1:06

**Comment by GoogleCodeExporter** [comment]
```
Patches welcome.
```

Original comment by `felipe.contreras` on 20 Jun 2009 at 10:48
- Changed title: **Make win32 installer ask for Pidgin installation if not found**
- Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
Original comment by `felipe.contreras` on 14 Dec 2009 at 12:29
- Changed state: **Accepted**
- Added labels: **Milestone-1.0.0**

**Comment by GoogleCodeExporter** [comment]
```
Targeting 0.2.0.
```

Original comment by `felipe.contreras` on 25 Jan 2010 at 3:51
- Added labels: **Milestone-0.2.0**
- Removed labels: **Milestone-1.0.0**

**Comment by GoogleCodeExporter** [comment]
```
In fact I just implemented this, will be 0.1.0-rc3.
```

Original comment by `felipe.contreras` on 27 Jan 2010 at 12:34
- Changed state: **Fixed**
- Added labels: **Milestone-0.1.0**
- Removed labels: **Milestone-0.2.0**

**Comment by GoogleCodeExporter** [comment]
```
Can somebody verify that this works on v0.1.0?
```

Original comment by `felipe.contreras` on 21 Jun 2010 at 11:33
